eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -839,6 +840,41 @@ mod tests {
         assert_eq!(finalized_slot, finalized_slot_1);
     }
 
+    #[test]
+    fn test_finality_light_client_update_correctness() {
+        const TREE_FINALITY_DEPTH: usize = 6;
+        const  TREE_FINALITY_INDEX: usize = 41;
```

Comment:
```suggestion
        const TREE_FINALITY_INDEX: usize = 41;
```

Reply:
Also, it will be good to move all Ethereum constants to one module including this one 
https://github.com/aurora-is-near/rainbow-bridge/blob/master/eth2near/eth2near-block-relay-rs/src/init_contract.rs#L13

Reply:
@karim-en should be fixed in the scope of a separate PR.